README.md

```diff
@@ -11,13 +11,19 @@ Use the Node-RED `Manage Palette` command or run the following in your Node-RED
 
 The `q-gate` node is similar to the `gate` node published as [node-red-contrib-simple-gate](https://flows.nodered.org/node/node-red-contrib-simple-gate) but with the added capability of queueing messages and releasing them when triggered. 
 
-The node will transmit the input message to its output when in the `open` state and block it when `closed`. In the `queueing` state, the input message is added to the end of the message queue, provided space is available.  Messages in the queue can be released (in the order received) either singly or the entire queue at once.
+The node will transmit the input message to its output when in the `open` state and block it when `closed`. In the `queueing` state, the input message is added to the end of the message queue, provided space is available.  Messages in the queue can be released (in the order received) either singly or the entire queue at once. Alternatively the oldest message in the queue can be sent (`peek`) without releasing it or the oldest message can be removed from the queue without sending it on (`drop`).
 
 The user can limit the size of the queue to prevent memory problems. Messages arriving when the queue is full are discarded by default, so that the queue contains the oldest messages. Since version 1.1.0, however, the user can select the `Keep newest messages` checkbox to have new messages added to the queue (at the tail), while discarding the oldest message (from the head), so that the queue contains the most recent messages. This feature makes it possible to retain only the latest message and deliver it on request, as shown in the example below.
 
 Messages with the user-defined topic `Control Topic` (set when the node is deployed) are not passed through but are used to control the state of the gate or the queue.
 
-Control messages can have values representing commands that change the state of the gate: `open`, `close`, `toggle`, `queue`, or `default`. Messages that control the queue are `trigger`, `flush`, and `reset`. The (case-insensitive) strings representing these commands are set by the user when the node is deployed. If a control message is received but not recognized, there is no output or change of state, and the node reports an error.
+Control messages can have values representing commands that change the state of the gate: `open`, `close`, `toggle`, `queue`, or `default`. Messages that control or monitor the queue are `trigger`, `flush`, `peek`, `drop`, `status`, and `reset`. The (case-insensitive) strings representing these commands are set by the user when the node is deployed. If a control message is received but not recognized, there is no output or change of state, and the node reports an error.
```

Comment:
This suggests that `status` operates on the `queueing` state. Doesn't it also refresh the status display for `open` and `closed` also?

Reply:
Yes, I am sure you are correct.  I haven't actually tested that though.
This is the first time I have been in this state with a PR, do I have to do anything or are you doing it?

Reply:
New experience for me too. I can't modify your PR, so the "correct" way is for you to fix it up until I am happy to merge it into the master. Still, you don't have to make all the changes, since I can do some after merging.

Reply:
At the bottom of ones where you have made a suggestion there is a Commit Changes button, if I click that does that make the changes to my fork and hence the PR.  I tried Google but didn't find a reference to this.
Then what does the Resolve conversation button that I also see do?

Reply:
I think the Commit Changes button merges my changes into your PR. I think I don't see the button because I can't use it. Resolve conversation just hides the discussion, like the one for q-gate.js further down the page.

Reply:
I have checked and the status command does work whether queueing or not.  Do you want me to tweak the words for that or do you want to phrase it yourself?

Reply:
I'll take care of it, thanks.